scripts/auth.ps1

```diff
@@ -0,0 +1,46 @@
+param (
+    [string]$server = "https://localhost:5001"
+)
+
+# Ignoring self-signed certificates
+add-type @"
+    using System.Net;
+    using System.Security.Cryptography.X509Certificates;
+    public class TrustAllCertsPolicy : ICertificatePolicy {
+        public bool CheckValidationResult(
+            ServicePoint srvPoint, X509Certificate certificate,
+            WebRequest request, int certificateProblem) {
+            return true;
+        }
+    }
+"@
+[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
+
+$username = Read-Host 'Please enter your username'
+$password = Read-Host 'Please enter your password' -AsSecureString
+$body = @{
+    Username=$username
+    Password=[Runtime.InteropServices.Marshal]::PtrToStringAuto([Runtime.InteropServices.Marshal]::SecureStringToBSTR($password))
+} | ConvertTo-Json
+$headers = @{
+    "Content-Type" = "application/json"
+}
+try
+{
+    $response = Invoke-WebRequest -Uri $server/api/account/login -Method POST -Body $body -Headers $headers
+}
+catch
+{
+    if (  $_.Exception.response.StatusCode -eq "Unauthorized" )
+    {
+        Write-Output "Incorrect username or password"

```

Comment:
Write-Error

Reply:
In my opinion Write-Warning will better look in this case.
![image](https://user-images.githubusercontent.com/38095999/102057155-27911200-3df6-11eb-9941-607e85ac5876.png)

Reply:
Sure, lets do it this way! 👍